operations/spackle.js

```diff
@@ -5,7 +5,61 @@ var fs = require('fs');
 var childProcess = require('child_process');
 var deltas = require('./deltas');
 
-var writtenFiles = [5, 2015, 2016, 2017, 2018, 2019].flatMap((year, i, arr) => {
```

Comment:
The change is not needed, but if it was, we’d use the array.prototype.flatmap package instead of inlining.